Merge POM changes, and other improvements.

JSerraDevOps opened the pull request:
This merge includes all the changes to implement POM (that were present in the Branch 3_4). Then it also includes some refactoring to eliminate the custom delay functions, and other code cleanups.

DerunChen committed:
feat: refactorice test4

DerunChen committed:
resolve: conflicts

DerunChen committed:
feat: refactorizacion del test 3

DerunChen committed:
feat: POM for all tests

DerunChen committed:
remove: unused imports

JSerraDevOps committed:
Merge branch 'Test3_4' into MergePOM

# Conflicts:
#	src/test/java/starter/Test3_4.java
#	src/test/java/starter/Test5_6.java

JSerraDevOps committed:
Merge POM implementation from Test3_4 branch.
Cleanup of some unused vars and imports.
Improve on some locators all locators move to the PageObjects.
Cleanup of tests files so that we have a file per test.

JSerraDevOps committed:
Merged ProductPage class into HomePage class.
Fixed tests 3 and 5. Delays custom function removed.
CleanedUp unused statements.

y23aa commented:
This merge includes all the changes to implement POM and the elimination of the custom delay functions, and the implementation of other code cleanups.

Unit6253 reviewed (approved):
The new implementation add what is required for test 3 and 4 and also correct the await problem

Unit6253 reviewed (approved):
The new code implementations correct the issues pointed out in fedback

Unit6253 reviewed (approved):
Improve POM execute the project as requested

y23aa commented:
All changes accept it per feedback's instructions

JSerraDevOps committed:
Fixed issue in Test1 and Test2 when the browser is not maximized in low screen resolutions.
Improved Test4 that failed when the cart was not drawn quick enough.

UB2023 committed:
- Se agregan los binarios de Mac tanto para CPU de Apple como para Intel.
- Se agregan mensajes que identifican el nombre del test, sistema operativo y fecha en el que se corren
- Se actualiza el fichero serenity.conf
